johnzon-jsonb/src/main/java/org/apache/johnzon/jsonb/JohnzonBuilder.java

```diff
@@ -131,9 +131,12 @@ public Jsonb build() {
                     return it;
                 }).orElse(false);
 
-        if (config.getProperty(JsonbConfig.FORMATTING).map(Boolean.class::cast).orElse(false)) {
-            builder.setPretty(true);
-        }
+        config.getProperty(JsonbConfig.FORMATTING).map(Boolean.class::cast).ifPresent(builder::setPretty);
+        config.getProperty(AbstractJsonFactory.BUFFER_STRATEGY).map(String::valueOf).ifPresent(builder::setBufferStrategy);
```

Comment:
Must use johnzon.name pattern IMHO for consistency and not leak jsonp impl properties since we support other jsonp impl and already used shortnames for other feature.

Reply:
JohnzonBuilder#generatorConfig() and JohnzonBuilder#readerConfig() also use org.apache.johnzon.* like properties, also exposing json-p impl specific properties, so I reused those. Should those maybe be changed too?

Reply:
Yep, it shouldn't have been.
Was ok-ish in standalone/ee because it was not in the main case where jsonp impl was loaded automatically so not that visible in terms of classloading and runtime but this breaks the module design (and OSGi metadata).
It is fine to copy these strings in the builder (inline if triggered cause jsonp != null for backward compat) and we should just use a Map<String, ?> for the main case (single property, for instance "johnzon.jsonp.readerfactory.properties" and "johnzon.jsonp.generator.properties" - <jsonbimpl>.<config area>.<component>.<property> pattern?).
Will enable to handle any jsonp impl configuration and future properties addition without anything more.